public/resources/js/comments.js

```diff
@@ -109,18 +120,23 @@ $(document).ready(function() {
                     <div class="comment-edit-container" data-comment-id="${comment.id}"></div>
                     <p>${comment.created_at}</p>
                     <div class="comment-actions">`;
-
         if (userId === comment.user_id) {
             html += `<button class="comments-edit" data-comment-id="${comment.id}">Edit</button>`;
             html += `<button class="comments-delete" data-comment-id="${comment.id}">Delete</button>`;
         }
-
+    
         if (userId !== null) {
             html += `<button class="comments-reply" data-comment-id="${comment.id}">Reply</button>`;
         }
-
+    
         html += `</div></div>`;
-
+    
+        const childComments = allComments.filter(c => c.parent_comment_id === comment.id);
+        childComments.forEach(childComment => {
+            html += generateCommentHTML(childComment, allComments, level + 1);
+        });
+    
+        html += `</div>`;
```

Comment:
instead of load everything at once and dynamic fix the structure it will better load just the father comments and make a action that say 'load replies' and then after click make an asynchronous query to load the child comments from that father